nservicebus/upgrades/support-policy.md

```diff
@@ -9,7 +9,7 @@ related:
 
 ## Supported Versions
 
-Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported, or any minor version released within the last year.
+Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported. For the NServiceBus package only (i.e. excluding downstream packages such as [transports](/nservicebus/transports/) and [persistence](/nservicebus/persistence/)), any minor version released within the last year is supported.
```

Comment:
@bording - regarding:

>The first paragraph is saying that all minors are supported in the 3 year period, which is incorrect.

Where ? In NServiceBus? I think what @kbaley and I had were more clear. Also for NServiceBus, what i understood was that the current policy is correct. i.e. not just the latest supported minor, but all minors for supported majors. In that sense, not sure what you have under NServiceBus is right.